src/Utils/StringHelper.php

```diff
@@ -30,20 +30,20 @@ public static function ensureMaxLength(?string $string, int $length): ?string
     public static function sanitizeDDE(string $text): string
     {
         // see #3189
-        if (\strlen($text) === 0) {
+        if ($text === '') {
             return $text;
         }
 
         $sanitize = false;
 
         if (\in_array($text[0], self::DDE_PAYLOADS)) {
             $sanitize = true;
-        } elseif (stripos($text, 'DDE') !== false) {
+        } elseif (\stripos($text, 'DDE') !== false) {
             $sanitize = true;
         }
 
         if ($sanitize) {
-            // trying to prevent fucking Microsoft "feature" DDE
+            // trying to prevent Microsoft "feature" DDE
```

Comment:
that is not bad wording, that is one of the the worst "features" that MS has ever produced in his "office" products. I hate it and wasted way too many precious hours of my life, because ... well read about DDE injection and you will fall from your chair.

Reply:
I know that MS has done some really eval things already but i would rather call it "horrible" or whatever but not "fu***"

Reply:
Yeah true, latter one is actually to good to bring it up in one sentence with Microsoft 🙈 👿

Reply:
Code should not be a place to express your feelings even though you "wasted way too many precious hours of my life" which I feel very sorry about.

Reply:
Why? This is my code.

Reply:
Sure, if you want to leave it there, you can tell me to change it back.